src/OpenSage.Game/Data/W3d/W3dVertexMappingType.cs

```diff
@@ -60,6 +60,77 @@ public enum W3dVertexMappingType : uint
         /// NOTE: even if you don't want to animate the bump matrix, you should use this mapper
         /// so that the matrix gets set up with the identity settings.
         /// </summary>
-        BumpEnv
+        BumpEnv,
+
+        Mask,
+        Stage1_Uv,
     }
+
+    /*
```

Comment:
I don't think we need any of this comment? We already have these constants defined elsewhere.